### ISSUE TITLE
Media type in Typo3 > 7

### ISSUE THREAD

**Issue by hweckhardt** [nitsan-technologies/ns_all_lightbox#3]
I cannot find partials in the releases 3.1.1 for other media elements than images. In the folders Typo3-7 (under each variant of the lightbox) i can find the partial MediaGallery.html, with 3 case values 3=audioType, 4=videoType and TRUE=imageType.

In the folders Typo3 only Media/Type/image.html with figure class = "image".

Does that mean that this extension only works for media type "image", and are you planning to extend the functionality again to the other media types as well?

**Comment by CDRO** [comment]
I think the issue here is that there are typoscript conditions about compatVersion that seem wrong to me.

The condition states `[compatVersion = 8.7.0]` which suggests that it's only compatible 8.7.0 ([but that does not seem to be the case either](https://docs.typo3.org/m/typo3/reference-typoscript/8.7/en-us/singlehtml/#compatversion)), but you have to ensure that the compatVersion ist set in the install tool.

AFAIK the Media/Type/Image.html Partial is used by all elements from the core that use an image in any way, so this might not be the culprit.

To sum up: try setting the compatVersion in the install tool and hopefully you get where you want to be.
